docs-src/go/how-to-schedule-a-workflow-in-go.md

```diff
@@ -0,0 +1,233 @@
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Start the schedule paused to demonstrate how to unpause a schedule
+schedule.Description.Schedule.State.Paused = true
+```
+
+<!--SNIPEND-->
+
+Pausing can also be done by using `Pause()` on `scheduleHandle`.
+
+```go
+scheduleHandle.Pause()
+```
+
+To unpause a Schedule, use `Unpause()` on `scheduleHandle`.
+
+<!--SNIPSTART samples-go-schedule-unpause-describe {"selectedLines": ["107-110"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+// Unpause schedule
+log.Println("Unpausing schedule", "ScheduleID", scheduleHandle.GetID())
+err = scheduleHandle.Unpause(ctx, client.ScheduleUnpauseOptions{})
+if err != nil {
+	log.Fatalln("Unable to unpause schedule", err)
+}
+```
+
+<!--SNIPEND-->
+
+## Trigger
+
+Triggering a Schedule immediately executes an Action defined in the Schedule.
+By default, `trigger` is subject to the Overlap Policy.
+
+To trigger a Scheduled Workflow Execution in Go, use `Trigger()` on `scheduleHandle`.
+Set desired changes in `ScheduleTriggerOptions`.
+
+<!--SNIPSTART samples-go-schedule-trigger {"selectedLines": ["55-57"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+	// Manually trigger the schedule once
+	log.Println("Manually triggering schedule", "ScheduleID", scheduleHandle.GetID())
+
+	err = scheduleHandle.Trigger(ctx, client.ScheduleTriggerOptions{
+		Overlap: enums.SCHEDULE_OVERLAP_POLICY_ALLOW_ALL,
+	})
+	if err != nil {
+		log.Fatalln("Unable to trigger schedule", err)
+	}
+```
+
+<!--SNIPEND-->
+
+## Update
+
+Updating a Schedule changes the configuration of existing Schedules.
+
+Use the `Update()` command to modify an existing Schedule.
+
+<!--SNIPSTART samples-go-schedule-update {"selectedLines": ["65-99"]}-->
+
+[schedule/starter/main.go](https://github.com/temporalio/samples-go/blob/master/schedule/starter/main.go)
+
+```go
+	// Update the schedule with a spec so it will run periodically,
+	log.Println("Updating schedule", "ScheduleID", scheduleHandle.GetID())
+	err = scheduleHandle.Update(ctx, client.ScheduleUpdateOptions{
+		DoUpdate: func(schedule client.ScheduleUpdateInput) (*client.ScheduleUpdate, error) {
+			schedule.Description.Schedule.Spec = &client.ScheduleSpec{
+				// Run the schedule at 5pm on Friday
+				Calendars: []client.ScheduleCalendarSpec{
+					{
+						Hour: []client.ScheduleRange{
+							{
+								Start: 17,
+							},
+						},
+						DayOfWeek: []client.ScheduleRange{
+							{
+								Start: 5,
+							},
+						},
+					},
+				},
+				// Run the schedule every 5s
+				Intervals: []client.ScheduleIntervalSpec{
+					{
+						Every: 5 * time.Second,
+					},
+				},
+			}
+			// Start the schedule paused to demonstrate how to unpause a schedule
```

Comment:
This is not starting (may have to go back and fix that comment in samples) and this isn't for demonstrating unpause

Reply:
https://github.com/temporalio/samples-go/blob/main/schedule/starter/main.go

Reply:
There's nothing else demonstrating. I'm working with what's provided, so this looks like a good time to go over the current code samples.

Reply:
:+1: may have to go back and fix that comment in samples

> There's nothing else demonstrating

Exactly, so should fix that comment

Reply:
Oh I didn't take it that way, you're good! Ideally we can get the code right from the outset, but if not, ok.